tests/decorators/test_async.py

```diff
@@ -118,3 +118,34 @@ async def expensive_function(**kwargs: Any) -> int:
         assert await expensive_function(a="a") == 1
 
     m_set.assert_called_with(mock.ANY, mock.ANY, time_to_live=timedelta(seconds=42), if_not_exists=mock.ANY)
+
+
+def sync_exclude(key_, value_):
+    return int(key_) + value_ < 40
+
+
+async def async_exclude(key_, value_):
+    return sync_exclude(key_, value_)
+
+
+@pytest.mark.parametrize(
```

Comment:
This looks somewhat excessive since both cases are now covered by `maybe_awaitable`, so you could shorten this to a simple inline lambda for `exclude`. You probably replicated `sync_ttl`/`async_ttl` and I'm going to refactor them anyway

Reply:
Agreed, I thought it would still add a bit of value to testing the API explicitly but i don't mind leaving it out